package.json

```diff
@@ -9,6 +9,7 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "antd": "^5.0.2",
```

Comment:
Hi! It seems that only installing a dependency will not be enough, is it possible to immediately check and import some component from there? will it work?
It seems to me that additional style imports are still needed. Please try to replace the button in App.tsx for changing the counter with a button from the antd library to be sure that the library works correctly on the project.

Reply:
Also required to add: ThemeProvider, icons, styles

https://ant.design/docs/react/customize-theme

https://github.com/ant-design/ant-design-dark-theme

Reply:
Counter Button changed,
Icons installed,
added ConfigProvider to main.tsx as an example used token with cyan-7 from base color pallete. 
according to documentation of v 5.0.2: 
"_Css files are no longer included in package. Since CSS-in-JS supports importing on demand, the original antd/dist/antd.css has also been abandoned. If you need to reset some basic styles, please import antd/dist/reset.css_"

Ant Design Dark Theme is not supported in v5, should we downgrade to v 4?

Reply:
Ant Design Dark Theme is supported in v5
Use Preset Algorithms (https://ant.design/docs/react/customize-theme)

```
import React from 'react';
import { ConfigProvider, Button, theme } from 'antd';

const App: React.FC = () => (
  <ConfigProvider
    theme={{
      algorithm: theme.darkAlgorithm,
    }}
  >
    <Button />
  </ConfigProvider>
);

export default App;
```